src/main/java/brandkon/product/Product.java

```diff
@@ -0,0 +1,58 @@
+package brandkon.product;
+
+import brandkon.brand.Brand;
+import brandkon.brand.BrandCategory;
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+public class Product {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String productName;
+    private int price;
+    private String imageUrl;
+    private int expirationDays;
+
+
+    @ManyToOne
+    private Brand brand;
+
+    public Product() {
```

Comment:
사소한 거긴 한데 우리가 사용할 일은 없는 생성자니까 `public`보다는 `protected`가 좀 더 코딩을 잘해 보이는 효과가..
```suggestion
    protected Product() {
```